javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -218,6 +218,23 @@ public static boolean isSelfAccess(final ExpressionTree tree) {
     return t;
   }
 
+  /**
+   * If the given tree is a parenthesized tree or cast tree, return the enclosed non-parenthesized,
```

Comment:
Perhaps, rather than introducing this new method, `withoutParens` should also strip away casts.